src/Diagnostics.ModelsAndUtils/Diagnostics.ModelsAndUtils.csproj

```diff
@@ -9,6 +9,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="CommonMark.NET" Version="0.15.1" />
```

Comment:
make sure this dll is part of the build package. If its not, we would need to update dotnet feed to incliude this package in dotnet package cache. (Few referenced packages are part of dotnet package cache)

Reply:
How do I check that?

Reply:
just run 'build' from root and check the runtime host package folder whether it has this dll or not. If its there, then all's good.

Reply:
It is there :)